README.md

```diff
@@ -10,18 +12,41 @@ This is the EnergyPlus Development Repository.  EnergyPlus™ is a whole buildin
  - Many users (and developers) of EnergyPlus are active on [Unmet Hours](https://unmethours.com/), so that's a great place to start if you have a question about EnergyPlus or building simulation.
  - For more in-depth, developer-driven support, please utilize the [EnergyPlus Helpdesk](http://energyplus.helpserve.com/).
 
+## Testing
+
+![](https://github.com/NREL/EnergyPlus/workflows/Custom%20Check/badge.svg) ![](https://github.com/NREL/EnergyPlus/workflows/Documentation/badge.svg) ![](https://github.com/NREL/EnergyPlus/workflows/CppCheck/badge.svg)
+
+Every commit and every release of EnergyPlus undergoes rigorous testing.
+The testing consists of building EnergyPlus, of course, then there are unit tests, integration tests, API tests, and regression tests.
+Since 2014, most of the testing has been performed by our bots ([Tik-Tok](https://github.com/nrel-bot), [Gort](https://github.com/nrel-bot-2), and [Marvin](https://github.com/nrel-bot-3)), using a fork of the [Decent CI](https://github.com/lefticus/decent_ci) continuous integration system.
+We are now adapting our efforts to use the Github Actions system to handle more of our testing processes.
+In the meantime, while Decent CI is still handling the regression and bulkier testing, results from Decent CI are still available on the testing [dashboard](http://nrel.github.io/EnergyPlusBuildResults/).
+
 ## Releases [![](https://img.shields.io/github/release/NREL/energyplus.svg?colorB=FF7300)](https://github.com/NREL/EnergyPlus/releases/latest)
 
+![](https://github.com/NREL/EnergyPlus/workflows/Windows%20Releases/badge.svg) ![](https://github.com/NREL/EnergyPlus/workflows/Mac%20Releases/badge.svg) ![](https://github.com/NREL/EnergyPlus/workflows/Linux%20Releases/badge.svg)
+
 EnergyPlus is released twice annually, usually in March and September.
 It is recommended all use of EnergyPlus is production workflows use these formal, public releases.
 Iteration **(pre-)releases** may be created during a development cycle, however users should generally avoid these, as input syntax may change which won't be supported by the major release version transition tools, and could require manual intervention to remedy.
 If an interim release is intended for active use by users, such as a bug-fix-only or performance-only re-release, it will be clearly specified on the release notes and a public announcement will accompany this type of release.
+Our releases are now built by Github Actions.
 
-## Documentation [![](https://img.shields.io/badge/docs-pdf-FF7300.svg)](https://energyplus.net/documentation) [![](https://img.shields.io/badge/docs-html-FF7300.svg)](http://bigladdersoftware.com/epx/docs/)
+## Core Documentation
 
 Program documentation is installed alongside the program, with the pdfs also available [online](https://energyplus.net/documentation).
 Big Ladder also produces html based documentation [online](http://bigladdersoftware.com/epx/docs/).
 
+## API Documentation
```

Comment:
Added API documentation section including a link to our brand new read the docs project.  It needs a lot of love, but it is a push in the right direction.